contributors/design-proposals/propagation.md

```diff
@@ -109,6 +125,12 @@ Opinion against this:
 
 1. (From @jonboulle) May cause cross-runtime compatibility issue.
 
+1. It's not possible to validate a pod + mount propagation. Mount propagation
```

Comment:
The PV node affinity is designed so that it could be used on other volume types, but right now it is behind the local volume feature gate.